impl/src/main/java/org/jboss/cdi/tck/tests/context/conversation/servlet/ServletConversationTest.java

```diff
@@ -184,6 +199,44 @@ public void testInvalidatingSessionDestroysConversation() throws Exception {
         }
     }
 
+    @Test
+    @SpecAssertion(section = CONVERSATION_CONTEXT_EE, id = "qa")
+    public void testInvalidatingSessionDestroysAllLongRunningConversations() throws Exception {
+        WebClient client = new WebClient();
+
+        // reset ActionSequence and begin two conversations
+        client.getPage(getPath("/resetSequence", null));
+        TextPage firstConversation = client.getPage(getPath("/begin", null));
+        String content = firstConversation.getContent();
+        String firstCid = getCid(content);
+        assertTrue(content.contains("message: Hello") && content.contains("transient: false"));
+        
```

Comment:
This 4 lines (maybe 5) above seem to appear quite often in this test. What about to extract them to some method? I think it would make the test little bit legible .